### PR TITLE
Bump ruby versions

### PR DESCRIPTION
**Pull request by sgringwe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ A gem for easily using read replicas.
 
 ## Supported Versions
 
-Knockoff supports Rails 4 and 5
+Knockoff supports Rails 4, 5 and 6
```
Comment (hoguer) [inline README.md:20]: nit: 4, 5, 6, and 7

File: .travis.yml [hunk 2]
```diff
@@ -2,7 +2,8 @@ language: ruby
 notifications:
   email: false
 rvm:
-  - 2.3.1
   - 2.4.0
```
Comment (hoguer) [inline .travis.yml:5]: guessing we don't want to bother bumping the minor versions of 2.4 and 2.5?

Comment (sgringwe) [reply .travis.yml:5]: i see us removing support for this in the near future, so i figured it's not worth messing with

**Review by hoguer — APPROVED** [review]
Looks good!

**Pull request merged**